### PR TITLE
Docs: experimental CLI docs

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
Documentation for #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -259,6 +259,37 @@ CustomApp.getInitialProps = async (ctx: AppContext) => {
 
 <br />
 
+# ⚗️ CLI (experimental)
+
+You can use `unleash [action] [options]` in you `package.json` `scripts` section, or with:
```
Comment (thomasheartman) [inline README.md:264]:
```suggestion
You can use `unleash [action] [options]` in your `package.json` `scripts` section, or with:
```

File: README.md [hunk 2]
```diff
@@ -259,6 +259,37 @@ CustomApp.getInitialProps = async (ctx: AppContext) => {
...
+
+```sh
+npx @unleash/nextjs
+```
+
+Use [Environment Variables](#environment-variables) (you needed: `UNLEASH_SERVER_API_URL`, `UNLEASH_SERVER_API_TOKEN`), or shorthand versions: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`.
```
Comment (thomasheartman) [inline README.md:270]: What do you use environment variables for? I don't understand why this sentence is telling me to do something.

```suggestion
Use [environment variables](#environment-variables) (you need: `UNLEASH_SERVER_API_URL`, `UNLEASH_SERVER_API_TOKEN` or shorthand versions: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`).
```

File: README.md [hunk 3]
```diff
@@ -259,6 +259,37 @@ CustomApp.getInitialProps = async (ctx: AppContext) => {
...
+
+Try it now
+
+```sh
+UNLEASH_URL=https://app.unleash-hosted.com/demo/api \
+UNLEASH_TOKEN=test-server:default.8a090f30679be7254af997864d66b86e44dcfc5291916adff72a0fb5 \
+npx @unleash/nextjs generate-types ./unleash.ts
+```
+
+<br/>
```
Comment (thomasheartman) [inline README.md:291]: I let a few of these slip through previously, but I personally don't like using manual breaks in the markup. Much like with HTML, I don't think the text should try and define how it's presented. So I suggest we remove this one. 

That said, I'm more than happy to let you make a case for why we should keep them if you think we should.

```suggestion
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Cool! This is very exciting 😄  Got some minor fixes and typos, plus one question that requires a little more work to address, I think.

File: README.md (hunk 2 above)
Comment (Tymek) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (thomasheartman) [reply README.md:283]: @Tymek I noticed you resolved this one and removed all the breaks. But it seems you might have forgotten this one? 😅

File: README.md [hunk 4]
```diff
@@ -257,7 +251,36 @@ CustomApp.getInitialProps = async (ctx: AppContext) => {
...
-<br />
+# ⚗️ CLI (experimental)
+
+You can use `unleash [action] [options]` in your `package.json` `scripts` section, or with:
+
+```sh
+npx @unleash/nextjs
+```
+
+CLI will attemp to read NextJS `.env` files. You need to set `UNLEASH_SERVER_API_URL` and `UNLEASH_SERVER_API_TOKEN` (see [environment variables](#environment-variables)). Another way of configuring the source of feature toggles is to pass environment variables directly, like in the example below. Available shorthand variables are: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`.
```
Comment (thomasheartman) [inline README.md:262]: I'm still not sure **why** I need to set these environment variables. What do they do? 

And also, what are the shorthand variables for? They're not mentioned in the "environment variables" section. Why not? Are they only for the CLI? 

`UNLEASH_APPNAME` is also different from `UNLEASH_APP_NAME`. Is that intentional or a typo?

```suggestion
For the CLI to work, the following [environment variables](#environment-variables) must be configured with appropriate values:

- `UNLEASH_SERVER_API_URL` 
- `UNLEASH_SERVER_API_TOKEN`

The CLI will attempt to environment values from any `.env` files if they're present. You can also set the variables directly when invoking the interface, as in the [CLI usage example](#example).
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Thank you 😄 I'm still not sure about that one section, though 🤔

File: README.md [hunk 5]
```diff
@@ -257,7 +249,39 @@ CustomApp.getInitialProps = async (ctx: AppContext) => {
...
+```sh
+npx @unleash/nextjs
+```
+
+For the CLI to work, the following [environment variables](#environment-variables) must be configured with appropriate values:
+
+- `UNLEASH_SERVER_API_URL`
+- `UNLEASH_SERVER_API_TOKEN`
+
+The CLI will attempt to environment values from any `.env` files if they're present. You can also set the variables directly when invoking the interface, as in the [CLI usage example](#example).
```
Comment (thomasheartman) [inline README.md:265]: Realized this was missing a word 😅 

```suggestion
The CLI will attempt to read environment values from any `.env` files if they're present. You can also set the variables directly when invoking the interface, as in the [CLI usage example](#example).
```

**Review by thomasheartman — APPROVED** [review]
Found a missing word that we should fix (my fault). Otherwise it looks fine to me, but the docs don't mention the shorthand options that they did previously. Is there a reason you took them out? 

(I didn't include them in my example because I don't know what they do, but I don't think you responded to that question for more info?)

File: README.md (hunk 5 above)
Comment (Tymek) [reply README.md:265]: I think I will drop shorthand ENV for simplicity.

**Pull request merged**